modules/samples/artifacts/CDCSample/CDCWithListeningMode.siddhi

```diff
@@ -27,22 +27,22 @@ Pre-requesites:
 Executing the Sample:
     1) Start the Siddhi application by clicking on 'Run'.
     2) If the Siddhi application starts successfully, the following message is shown on the console
-        * ReceiveChangeDataFromDatabase.siddhi - Started Successfully!
+        * CDCWithListeningMode.siddhi - Started Successfully!
 Note:
     If you want to edit this application while it's running, stop the application, make your edits and save the application, and then start it again. 
 
 Testing the Sample:
     Insert data to the created table.
         >> insert into SweetProductionTable values('chocolate',100.0);
     Observe the inserted data is logged on the console.
-    ReceiveChangeDataFromDatabase : logStream : Event{timestamp=1541486685789, data=[chocolate, 100.0], isExpired=false}
+    CDCWithListeningMode : logStream : Event{timestamp=1541486685789, data=[chocolate, 100.0], isExpired=false}
     
     Optional: Test for operation = 'update' and operation = 'delete' also.
     Note:   delete operation events will include keys: before_name, before_amount.
             update operation events will include keys: before_name, name, before_amount, amount.
 */
 
-@App:name('ReceiveChangeDataFromDatabase')
+@App:name('CDCWithListeningMode')
```

Comment:
Shall we move the @App:name  and @App:description to the top of the file and then have the purpose.  Please change in the other sample as well